### PR TITLE
Feature/pp9 10020 simplify selection system

### PR DESCRIPTION
**Pull request by rjs-picturepark**

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/list-browser/list-browser.component.ts [hunk 1]
```diff
@@ -186,33 +183,38 @@ export class ListBrowserComponent extends BaseBrowserComponent<ListItem> impleme
...
 
-    this.selectedItems = this.selection.selected;
-    this.selectedItemsChange.emit(this.selection.selected);
+  }
+
+  public isRowSelected(row: any) {
+    return this.selectionService.getById(row._refId) ? true : false;
   }
 
   public toggle(row: any) {
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/list-browser/list-browser.component.ts]: Change toggle to selection service

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/list-browser/list-browser.component.ts [hunk 2]
```diff
@@ -186,33 +183,38 @@ export class ListBrowserComponent extends BaseBrowserComponent<ListItem> impleme
...
+    }
   }
 
   /** The label for the checkbox on the passed row */
   public checkboxLabel(row?: any): string {
     if (!row) {
       return `${this.isAllSelected() ? 'select' : 'deselect'} all`;
     }
-    return `${this.selection.isSelected(row) ? 'deselect' : 'select'} row ${row.position + 1}`;
+    return `${this.selectionService.getById(row._refId) ? 'deselect' : 'select'} row ${row.position + 1}`;
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/list-browser/list-browser.component.ts]: use is row selected method

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/list-items/list/list.component.ts [hunk 3]
```diff
@@ -38,7 +38,8 @@ export class ListComponent implements OnInit, OnDestroy {
   public schemaDetail: SchemaDetail;
   public schema: Observable<SchemaDetail>;
   public schemaId: string;
-  public selectedItems: string[];
+  public selectedItems;
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/list-items/list/list.component.ts]: Add type

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/browser-base/browser-base.component.ts [hunk 4]
```diff
@@ -220,30 +220,30 @@ export abstract class BaseBrowserComponent<TEntity extends IEntityBase> extends
             this.lastSelectedIndex = index;
 
             if (itemModel.isSelected === true) {
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/browser-base/browser-base.component.ts]: @rjs-picturepark can we use selectionService.toggle() here?

**Pull request merged**